drivers/sensor/lis2dw12/lis2dw12.c

```diff
@@ -236,6 +236,20 @@ static int lis2dw12_set_power_mode(const struct device *dev,
 	return lis2dw12_write_reg(ctx, LIS2DW12_CTRL1, &regval, 1);
 }
 
+static int lis2dw12_set_low_noise(const struct device *dev,
+				  bool low_noise)
+{
+	const struct lis2dw12_device_config *cfg = dev->config;
+	stmdev_ctx_t *ctx = (stmdev_ctx_t *)&cfg->ctx;
+	lis2dw12_ctrl6_t ctrl6;
+
+	if (lis2dw12_read_reg(ctx, LIS2DW12_CTRL6, (uint8_t *)&ctrl6, 1)) {
+		return -EIO;
+	}
```

Comment:
please propagate error codes if you're not handling them,

```
ret = f();
if (ret < 0) {
    return ret;
}
```

applies to others

Reply:
I just mirrored what the rest of the driver was doing.
I'll refactor my part.